plugins/labhub.py

```diff
@@ -326,3 +328,50 @@ def eligible(user, iss):
                 yield ('The issue is already assigned to someone. Please '
                        'check if the assignee is still working on the issue, '
                        'if not, you should ask for reassignment.')
+
+    @staticmethod
+    def community_state(pr_count: dict):
```

Comment:
Is this overall or just for some timeperiod?

Reply:
Oh I see in the past x hours is it? Depending on the input?

Reply:
Yes, depending on the hours given in the command. `corobo pr stats 10hrs`, we consider last 10 hours

Reply:
Yes, depending on the hours given in the command. `corobo pr stats 10hrs`, we consider last 10 hours